changes/4796.added

```diff
@@ -0,0 +1,2 @@
+Added InterfaceRedundancyGroupAssociation to GraphQL.
+Added try...except... block to handle the `AttributeError` raised by `get_absolute_url()` when a valid absolute url is not found.
```

Comment:
Not sure I see the connection between these two items. Since as you mentioned the try/except logic may be needed in `develop`, is there a reason it shouldn't just be part of a separate PR that can bubble up to `next` in due time?

Reply:
Yeah confirmed the error in develop as well, will just open a PR in `develop`